src/components/Display.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Display.css';
 
-function Display() {
+function Display({ result }) {
   return (
     <div className="Display" id="button-panel">
-      <p> Calculator App: Hello world!</p>
+      <p>
+        {result}
+      </p>
     </div>
   );
 }
 
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
```

Comment:
- [ ] [OPTIONAL] The default value of the result propr should be '0', for when no result was passed yet.